src/components/Title/Title.tsx

```diff
@@ -2,29 +2,23 @@ import React from 'react';
 import {Button, Icon, Text} from '@gravity-ui/uikit';
 import {Xmark} from '@gravity-ui/icons';
 import {block} from '../utils/cn';
+import i18n from './i18n';
 
 import './Title.scss';
 
 const b = block('title');
 
-type TitleDictKeys = 'close';
-type TitleDict = Record<TitleDictKeys, string>;
-
-const defaultDict: TitleDict = {
-    close: 'Close',
-};
-
 interface TitleProps {
     hasSeparator?: boolean;
-    dict?: TitleDict;
+    closeTitle?: string;
     closeIconSize?: number;
     onClose?: () => void;
 }
 export const Title: React.FC<React.PropsWithChildren<TitleProps>> = ({
     children,
     closeIconSize = 23,
     hasSeparator,
-    dict = defaultDict,
+    closeTitle = i18n('button_close'),
```

Comment:
In such a case, will switching the language update the default value?

Reply:
I'll check